rskj-core/src/main/java/co/rsk/peg/federation/FederationChangeResponseCode.java

```diff
@@ -6,8 +6,7 @@ public enum FederationChangeResponseCode {
     EXISTING_FEDERATION_AWAITING_ACTIVATION(-2),
     RETIRING_FEDERATION_ALREADY_EXISTS(-3),
     PROPOSED_FEDERATION_ALREADY_EXISTS(-4),
-    PENDING_FEDERATION_NON_EXISTENT(-1),
-    RETIRING_FEDERATION_NON_EXISTENT(-1),
+    FEDERATION_NON_EXISTENT(-1),
```

Comment:
mmm i won't do this, we should aim to have different codes in the future, not to unify them

Reply:
https://github.com/rsksmart/rskj/pull/2782#discussion_r1784991712